seedlet-blocks/block-templates/singular.html

```diff
@@ -1,13 +1,13 @@
-<!-- wp:group {"align":"full","className":"site-header"} -->
+<!-- wp:group {"align":"full","className":"site-header","tagName":"header"} -->
 <div class="wp-block-group alignfull site-header"><div class="wp-block-group__inner-container"><!-- wp:template-part {"slug":"header","theme":"seedlet-blocks"} /--></div></div>
 <!-- /wp:group -->
 
-<!-- wp:group {"align":"full","className":"page-title"} -->
-<div class="wp-block-group alignfull page-title"><div class="wp-block-group__inner-container"><!-- wp:post-title /--></div></div>
+<!-- wp:group {"align":"full","className":"site-content","tagName":"main"} -->
+<div class="wp-block-group alignfull site-content"><div class="wp-block-group__inner-container">
+<!-- wp:post-title /-->
```

Comment:
When wrapped in a alignfull group block, the post title shows up for me like this: 

<img width="1650" alt="Screen Shot 2020-06-04 at 2 39 19 PM" src="https://user-images.githubusercontent.com/5375500/83797825-36da1a80-a671-11ea-8303-ec75967c8c4f.png">

Reply:
Hmm weird. I pushed an update in 490dc73, but it seems like it should be unnecessary. 🤔

Reply:
Nevermind, I figured it out — Seedlet's standard wide/full styles only apply themselves to blocks inside of `.entry-content`, so they're not active here. We should probably change that in Seedlet if we can, since it'll make things way easier for full-site editing. For now, I've duplicated those styles here in the child theme.